examples/common/scheduled_gc_callback.py

```diff
@@ -3,10 +3,19 @@
 
 import gc
 
+import torch
 from composer.core import Callback, State
 from composer.loggers import Logger
 
 
+# from: https://github.com/BlackHC/toma/blob/10cfe70efaba59ea669c50c0060cfddef65d0b16/toma/torch_cuda_memory.py#L10
```

Comment:
Idk if you really need to credit this, it's pretty common code I think

Reply:
ok